pkg/portal/ssh/proxy.go

```diff
@@ -281,8 +281,8 @@ func (s *ssh) newChannel(ctx context.Context, accessLog *logrus.Entry, nc crypto
 	defer recover.Panic(s.log)
 
 	ch2, rs2, err := downstreamConn.OpenChannel(nc.ChannelType(), nc.ExtraData())
-	if err, ok := err.(*cryptossh.OpenChannelError); ok {
-		return nc.Reject(err.Reason, err.Message)
+	if errAsOpenChannel, ok := err.(*cryptossh.OpenChannelError); ok {
+		return nc.Reject(errAsOpenChannel.Reason, errAsOpenChannel.Message)
```

Comment:
This seems like a weird addition, was it meant for this PR? I don't hate it, just asking.

Reply:
yeah it is a leftover. I also had a golangci-lint update and it was complaining about it then.